config.json

```diff
@@ -1308,32 +1308,32 @@
       {
         "groupId": "com.google.mlkit",
         "artifactId": "smart-reply",
-        "version": "17.0.0",
-        "nugetVersion": "117.0.0.2",
+        "version": "17.0.1",
+        "nugetVersion": "117.0.1",
         "nugetId": "Xamarin.Google.MLKit.SmartReply",
         "dependencyOnly": false
       },
       {
         "groupId": "com.google.mlkit",
         "artifactId": "translate",
         "version": "17.0.0",
-        "nugetVersion": "117.0.0.2",
+        "nugetVersion": "117.0.0",
```

Comment:
Not sure what this change is.  If it needs to bump it needs to be `117.0.0.3`.  If it doesn't need to bump then is should stay `117.0.0.2`.